state/observable/Routable.js

```diff
@@ -0,0 +1,40 @@
+export const Routable = superclass => class extends superclass {
+	constructor({ id, token, state }) {
+		super({ id, token, state });
+		this._routes = new Map();
+	}
+
+	get childState() {
+		return this._childState;
+	}
+
+	set childState(state) {
+		this._childState = state;
+	}
+
+	/**
+	 * @param {Object} observer
+	 * @param {string} property
+	 * @param {} route
```

Comment:
JsDoc comments are a little weird with destructured parameters. This is the standard:
```suggestion
	 * @param {Object} obj
	 * @param {Object} obj.route - An object containing information on the route to go through
	 * @param {function} obj.method - A function that will mutate the value before setting the property 
```

Reply:
This makes sense. Will update.